src/main/java/cs/ualberta/cmput402/tictactoe/Scoreboard.java

```diff
@@ -0,0 +1,47 @@
+package cs.ualberta.cmput402.tictactoe;
+
+import cs.ualberta.cmput402.tictactoe.board.Board.Player;
+
+/**
+ * ScoreBoard
+ */
+public class Scoreboard {
+    private int tie = 0;
+    private int x_win = 0;
+    private int x_lose = 0;
+    private int o_win = 0;
+    private int o_lose = 0;
```

Comment:
Desired/Advised: the variable x_lose , o_lose are redundant since x_lose is equivalent to o_win and o_lose is equivalent to x_win. You should remove them for code simplicity.